app/assets/javascripts/components/overview/overview_stats.jsx

```diff
@@ -136,6 +136,8 @@ const OverviewStats = ({ course }) => {
         stat={course.view_count}
         statMsg={I18n.t(`metrics.${ArticleUtils.projectSuffix(course.home_wiki.project, 'view_count_description')}`)}
         renderZero={false}
+        info={I18n.t(`metrics.${ArticleUtils.projectSuffix(course.home_wiki.project, 'view_count_doc')}`)}
```

Comment:
This seems like it will break for Wikidata, since there's no corresponding message for when that suffix is present?

Reply:
That's true. I can create a:

view_count_doc_wikidata: This is the estimated number of views based on a 30-day average for each item, through the most recent stats update. Views may decrease if the updated average is lower than previous counts.

Is this text fine?